CONTRIBUTORS_QUICK_START.rst

```diff
@@ -544,6 +551,48 @@ Setting up Debug
                alt="Add environment variable pycharm">
         </div>
 
+- Now Debug an example dag and view the entries in tables such as ``dag_run, xcom`` etc in mysql workbench.
+
+B. Using Visual Studio Code
+
+- In Visual Studio Code open airflow project, directory ``\files\dags`` of local machine is by default mounted to docker
+  machine when breeze airflow is started. So any DAG file present in this directory will be picked automatically by
+  scheduler running in docker machine and same can be seen on ``http://127.0.0.1:28080``.
+
+- Copy any example DAG present in airflow project's ``\airflow\example_dags`` directory to ``\files\dags\``.
+
+- Add main block at the end of your DAG file to make it runnable. It will run a back_fill job:
```

Comment:
```suggestion
- Add a `__main__` block at the end of your DAG file to make it runnable. It will run a `back_fill` job:
```

Reply:
Double backticks for inline code in rst files.